src/node/psbt.cpp

```diff
@@ -79,9 +84,16 @@ PSBTAnalysis AnalyzePSBT(PartiallySignedTransaction psbtx)
         // Get the output amount
         CAmount out_amt = std::accumulate(psbtx.tx->vout.begin(), psbtx.tx->vout.end(), CAmount(0),
             [](CAmount a, const CTxOut& b) {
+                if (!MoneyRange(a) || !MoneyRange(b.nValue) || !MoneyRange(a + b.nValue)) {
+                    return CAmount(-1);
+                }
```

Comment:
Are you sure this is correct? This will reset the intermediate result to -1, and then accumulate all remaining values.

Reply:
-1 will fail MoneyRange so later values that go through this will also always return -1.

Reply:
Correct. Thx.